src/queries/models/channels/getChannelDetailsModelCreator.ts

```diff
@@ -0,0 +1,3 @@
+import { ChannelDetailsResponse } from 'queries/types';
+
+export const getChannelDetailsModelCreator = (response: ChannelDetailsResponse): ChannelDetailsResponse => response;
```

Comment:
@abhishek-01k we are not changing/re-modelling the data then why have this function or file?

Reply:
@mishramonalisha76 Can you see this since you wrote this func